### DISCUSSION TITLE
Multiple RDM6300 on one ESP32

### DISCUSSION THREAD

**Discussion by eckinga** [arendst/Tasmota#16057] [Q&A] [answered]
Hello,

i want to add more than one RDM6300 Rfid reader to one ESP32 and came from esphome where it seems not to be possible. So i just discovered that tasmota also supports the RDM6300 and gave it a try.
I connected one to GPIO3 and the other to GPIO16. In config it just activated and tested the one on gpio3 and all worked fine. then only the one on gpio16 and also worked fine. but if i activate both only the one on gpio3 works.

is there a way to get both working? and if its possible is the mqtt message seperated for both readers? 

thanks
Christian

**Comment by sfromis** [comment, marked as answer]
No, you can generally not expect it to work if you assign same component to more than one pin, unless it has a index dropdown to the right when configuring module or template. And this is not the case for RDM6300, the driver looks for the one pin configured as RDM6300 RX, and looks no further. You can get it working by using two ESP32 :wink:

**Comment by eckinga** [comment]
Hello sfromis,
thanks for reply. Then it would be nice if component would have a index and support multiple reader. One per ESP is not very effective. So I'm trying my luck myself with Arduino IDE.
